posts/2023-10-13-registrar-transition-update.md

```diff
@@ -9,4 +9,76 @@ tags:
+
+### Reducing paperwork
+
+Government organizations at all levels in the U.S. are eligible for a .gov domain. This includes cities, towns, counties, election offices, states, tribal governments, courts, federal agencies, and more. 
+
+One important change is to a part of government work that we're excited to reduce: *paperwork*. The current request process requires a signature, typically from the senior elected or highest-ranking official in your organization. For many governments, this inflexibility led to unnecessary bureaucracy that caused delays. **The new request process will be fully digital**, won't require a physical signature, and requests can typically be approved by an individual with significant, executive responsibility in your organization, like a senior technology officer or chief administrative official.
+
+## How these changes impact governments 
+
+### *If you want a .gov domain*
+
+- Though we won't accept new requests for a time, you can **prepare for reopening** by reviewing the [new process to request a .gov domain](../../domains/before/) on the beta version of our new .gov website. The work-in-progress site will help you learn about .gov and why governments should use it. 
+
+### *If you requested a .gov domain*
+
+- If you've submitted a request to us before today, it will be reviewed before the move to our new registrar. **You don't need to do anything else**, though we may reach out if we have additional questions.
+
+### *If you already have a .gov domain*
+
+- You'll need a [Login.gov account](https://www.login.gov/create-an-account/){.usa-link--external} that is linked to the same email address you're using in our current system to access the new registrar. If you manage a .gov domain but didn't receive an email from us about these changes, [let us know](../../contact) (you can also [read the email online](../2023-transition-email/#what-other-changes-impact-my-organization)).
+
+- Our contact details are changing. We'll begin using <help@get.gov> for email correspondence and will end phone support in November 2023.
+
+- Current domains will be automatically extended by a year and we will no longer perform certificate validation. See our last email for [more details](../2023-transition-email/#what-other-changes-impact-my-organization).
+
+- - -
+
+## What we heard from you
+
+Thank you for your questions and comments! Here are responses to what we heard.
+
+* [Does this mean that all .gov websites will shut down for three months?](#does-this-mean-that-all-gov-websites-will-shut-down-for-three-months)
+* [Can we launch new subdomains on .gov sites?](#can-we-launch-new-subdomains-on-gov-sites)
+* [What if some .gov domain managers don't create Login.gov accounts?](#what-if-some-gov-domain-managers-dont-create-logingov-accounts)
```

Comment:
The anchors changed in 11ty (the ID includes the question mark and percent encodes it `%3F`). Can you take a quick look at these?

Reply:
Fixed; sorry about that!